patroni/postgresql.py

```diff
@@ -100,9 +100,8 @@ def __init__(self, config):
         self._postgresql_conf = os.path.join(self._data_dir, config_base_name + '.conf')
         self._postgresql_base_conf_name = config_base_name + '.base.conf'
         self._postgresql_base_conf = os.path.join(self._data_dir, self._postgresql_base_conf_name)
+        self._postgresql_custom_conf = config.get('custom_conf')
```

Comment:
I think you need to convert it into 'property' as well, otherwise it can't be changed dynamically...